desktop-app/app/utils/browserSync.js

```diff
@@ -1,15 +1,38 @@
-const browserSync = require('browser-sync').create();
+const browserSyncEmbed = require('browser-sync').create('embed');
 
 import {BROWSER_SYNC_VERSION} from '../constants/browserSync';
 
+let filesWatcher;
+let cssWatcher;
 export async function initBrowserSync() {
-  if (!browserSync.active) {
+  if (!browserSyncEmbed.active) {
     await initInstance();
   }
 }
 
+export function watchFiles(fileDir) {
+  filesWatcher = browserSyncEmbed
+    .watch([`${fileDir}/*.html`, `${fileDir}/**/**.js`])
```

Comment:
@esprush 
If we are watching all `*.js` and all `*.css` in all sub folders, why not to do the same with `*.html`?

Reply:
I think we should only watch the currently loaded `.html` file, as the other HTML files are not related to the current page.

Reply:
@esprush  @jjavierdguezas Merging this now. Let's address any concerns in a new PR.